src/lib/Microsoft.Health.Fhir.Ingest.Template/CalculatedFunctionContentTemplateExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using EnsureThat;
+
+namespace Microsoft.Health.Fhir.Ingest.Template
+{
+    public static class CalculatedFunctionContentTemplateExtensions
+    {
+        /// <summary>
+        /// Iterates over all expressions belonging to the content template. If the expression language has not been set it will be set
+        /// to default language of the content template.
+        /// </summary>
+        /// <param name="template">The template to retrieve expressions from</param>
+        public static void EnsureExpressionLanguageIsSet(this CalculatedFunctionContentTemplate template)
```

Comment:
>void [](http://example.com/codeflow?start=22&length=4)

Suggestion: Return the CalculatedFunctionContentTemplate.  This will support chaining functions together in fluent like style.